L/LibTracyClient/LibTracyClient@0.9.0/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libTracyClient"
```

Comment:
Needs to have the same name as the directory.

Reply:
Hmm, I was trying to follow the convention from [LibUnwind](https://github.com/JuliaPackaging/Yggdrasil/blob/master/L/LibUnwind/LibUnwind%401.5.0/build_tarballs.jl)

At first I thought you were just referring to the capitalization, but I see now that it still fails. Do I need to include the "@0.9.0" after all?

Reply:
This name should match _exactly_, including capitalisation, the name of the directory the file is in, excluding whatever comes after the at-sign

Reply:
Ah in that case I think is correct now

Reply:
Uhm, yeah, I think there may be an error in the new pipeline because it's looking for the files including the at-sign, that part should be ignored